ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -168,6 +169,29 @@
         .withAuthFilePath(makeRelativePathAbsolute(table.getString("auth-file")))
         .withTimeout(table.getLong("timeout"));
 
+    final Boolean tlsEnabled = table.getOptionalBoolean("tls-enabled").orElse(true);
+    builder.withTlsEnabled(tlsEnabled);
+    if (tlsEnabled) {
+      final Optional<String> tlsTrustStoreFile = table.getOptionalString("tls-truststore-file");
+      final Optional<String> tlsTrustStorePasswordFile =
+          table.getOptionalString("tls-truststore-password-file");
```

Comment:
probably worth putting the password directly into the toml file rather than redirecting via another file (which is (sort of) how we did the Azure Secret)

Reply:
hmmm .... I would need to re-visit it as it may require a slight design change.

Reply:
we don't need password file anymore. resolved.